rasa_vier_cvg/cvg.py

```diff
@@ -253,15 +254,26 @@ def from_credentials(cls, credentials: Optional[Dict[Text, Any]]) -> InputChanne
         else:
             blocking_endpoints = bool(blocking_endpoints)
 
-        logger.info(f"Creating input with: token={'*' * len(token)} proxy={proxy} start_intent={start_intent} blocking_endpoints={blocking_endpoints}")
-        return cls(token, start_intent, proxy, blocking_endpoints)
+        ignore_messages_when_busy = credentials.get("ignore_messages_when_busy")
+        if ignore_messages_when_busy is None:
+            ignore_messages_when_busy = False
+        else:
+            ignore_messages_when_busy = bool(ignore_messages_when_busy)
+
+# There must be some way to access the tracker. But it is probably going to be private anyway...
+
 
-    def __init__(self, token: Text, start_intent: Text, proxy: Optional[Text], blocking_endpoints: bool) -> None:
+        logger.info(f"Creating input with: token={'*' * len(token)} proxy={proxy} start_intent={start_intent} blocking_endpoints={blocking_endpoints} ignore_messages_when_busy={ignore_messages_when_busy}")
+        return cls(token, start_intent, proxy, blocking_endpoints, ignore_messages_when_busy)
+
+    def __init__(self, token: Text, start_intent: Text, proxy: Optional[Text], blocking_endpoints: bool, ignore_messages_when_busy: bool) -> None:
         self.callback = None
         self.expected_authorization_header_value = f"Bearer {token}"
         self.proxy = proxy
         self.start_intent = start_intent
         self.blocking_endpoints = blocking_endpoints
+        self.ignore_messages_when_busy = ignore_messages_when_busy
+        self.ignore_messages_for = [str]
```

Comment:
declare it similar to the other fields and document the fact that this effectively prevents clustering.